CHANGELOG.md

```diff
@@ -1,5 +1,262 @@
 # Changelog
 
+## [0.58.0]
+
+### Added
+
+- Add end point for initializecore ([9687090](https://github.com/facebook/react-native/commit/9687090) by [@ejanzer](https://github.com/ejanzer))
+- Add tracing for loading .so files during startup ([b3b6749](https://github.com/facebook/react-native/commit/b3b6749) by [@axe-fb](https://github.com/axe-fb))
+- Added lock around RN module initialization to fix crash ([6770b53](https://github.com/facebook/react-native/commit/6770b53) by [@PeteTheHeat](https://github.com/PeteTheHeat))
+- Added locking around RN bridge cxx module registry to avoid crash ([1c31919](https://github.com/facebook/react-native/commit/1c31919) by [@PeteTheHeat](https://github.com/PeteTheHeat))
+- Add Yoga JNI bindings to libcoldstart ([2a8f6c3](https://github.com/facebook/react-native/commit/2a8f6c3) by [@davidaurelio](https://github.com/davidaurelio))
+- Adds support for `publicPath` to enable serving assets from different locations. (#299) ([0b31496](https://github.com/facebook/react-native/commit/0b31496) by [@gdborton](https://github.com/gdborton))
+- Add another guard to lazilyLoadView ([d7865eb](https://github.com/facebook/react-native/commit/d7865eb) by [@mmmulani](https://github.com/mmmulani))
+- Add a marker to indicate when JS thread priority is lowered ([d4aef08](https://github.com/facebook/react-native/commit/d4aef08) by [@axe-fb](https://github.com/axe-fb))
+- added functionality using which child node can tell parent node to use it as a reference baseline ([6eb5bd3](https://github.com/facebook/react-native/commit/6eb5bd3) by [@SidharthGuglani](https://github.com/SidharthGuglani))
```

Comment:
It is unclear to me whether this changes any public APIs for React Native or if it is internal to Yoga / RN's usage of Yoga. cc @sidhartguglani

Reply:
I think it's just RN's usage of Yoga, based on how other changes to these classes have been treated.